lesson11_list_comprehension/task2.py

```diff
@@ -0,0 +1,13 @@
+def task2(a):
+    digits = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Comment:
Make a function random by creating an array of integers of a given length.
```
makeIntArray(Size in) -> array of random int of a given size `in`
```
Put this function into `helpers.helper`, so you can reuse it next time

Reply:
Make enother function that creates an array of a given size but values in a given range
```
makeIntCustomArray(Size int, from int, step int) -> array of a given size, values starts `from` and filled with the `step`
```